### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by b-mi** [juan-manuel-alberro/angular-sweetalert2#2]
Installed angular wrapper, I use angular 4, and got this error:

ERROR TypeError: myWindow.Sweetalert2 is not a function
    at SweetAlertService.webpackJsonp.../../../../angular-sweetalert-service/js/service/sweetalert2.service.js.SweetAlertService.confirm (sweetalert2.service.js:25)

In component code I used
```
import { SweetAlertService } from 'angular-sweetalert-service';
not
import { SweetAlertService } from 'angular-sweetalert'; // this not work
```

Then, command from your page end with error:
```
        this.alertService.confirm({
            title: 'Delete account?'
          })
          .then(() => {
            this.alertService.success({
              title: 'Account deleted'
            });
          })
          .catch(() => console.log('canceled'));
```

**Comment by juan-manuel-alberro** [comment]
Hello @b-mi 
I'll take a look, maybe a new version was released and is not backward compatible.

**Comment by tunathoni** [comment]
Hello @juan-manuel-alberro i also get an error like this, is it was resolved ? or still not yet ?

**Comment by sangwin** [comment]
+1. Not working at all

**Comment by jeffreypolk** [comment]
You need to manually include sweetalert2 for this to work.

**Comment by b-mi** [comment]
@jeffreypolk - how exactly include sweetalert2 'manually '?  I use instructions for install from author -  but it not work. Did you try your solution on angular 4?

**Comment by jeffreypolk** [comment]
Yes using Angular4.  Not sure if it's the "right" way, but I just included sweetalert2 CDN js and css references in my index.html

**Comment by sokphea-chea** [comment]
I have the same problem like b-mi, and I think the problem come from webpack compile.

**Comment by asfo** [comment]
You can fix it by adding:

`"../node_modules/sweetalert2/dist/sweetalert2.min.js"`

And 

`"../node_modules/sweetalert2/dist/sweetalert2.min.css",`

To your angular-cli.json (in scripts and styles section obviously) :) and it works.

**Comment by jorgelub12** [comment]
Dont really work, There is no dist folder

**Comment by PixeliGer** [comment]
@b-mi I found that _'manual installation'_ means installing and adding SweetAlert2 in `angular.cli.json` 

**First Install SweetAlert2 vía npm**
`npm install --save sweetalert2`


**In your `angular.cli.json` include the css and js files**

```
"styles": [
        ...
        "../node_modules/sweetalert2/dist/sweetalert2.min.css",
        ...
      ],
"scripts": [
          ...
          "../node_modules/sweetalert2/dist/sweetalert2.min.js",
          ...
      ],
```

Now run` ng-serve` again, and `angular-sweetalert-service` will work without problems.



###Note
Im still with the webpack warning about compatibility
